src/app/types/Product.ts

```diff
@@ -0,0 +1,18 @@
+export interface Product {
+  id: string;
+  images: string[]; // Maximum of 3 images come on be serious
```

Comment:
Anecdotally I'm pretty sure 3 images for a product is a lowball (image carousels draw highest engagement on IG, articles generally suggest slightly higher averages of like 5-15 max images perform better for customer engagement on product pages, etc.), but also do not think it matters rn LMAO, just one to think about for future maybe / if it impacts what you're planning for BE at all